src/resources/postcss/my-tickets/_orders.pcss

```diff
@@ -4,26 +4,45 @@
  * @since 5.2.0
  */
 
+ .tribe-tickets__tickets-page-wrapper {
```

Comment:
Should this be `tribe-tickets__my-tickets-page-wrapper`?

Reply:
This was an old, existing class. I just added some styling to the element for the my tickets page.